eng/lintingconfigs/revapi/track2/revapi.json

```diff
@@ -85,7 +85,62 @@
   {
     "extension": "ignored-jackson-databind-removal",
     "configuration": {
-      "enabled": true
+      "enabled": true,
+      "ignoredPackages": {
```

Comment:
Why are all these packages ignored?

Reply:
This was making the RevApi extension customizable through configuration. This is just suppressing Jackson Annotation removals from those packages when we migrated to azure-json.